pkg/capacity/capacity.go

```diff
@@ -231,6 +231,7 @@ func (c *Controller) Run(ctx context.Context, threadiness int) {
 	klog.Info("Starting Capacity Controller")
 	defer c.queue.ShutDown()
 
+	go c.topologyInformer.Run(ctx)
```

Comment:
This looks like the wrong place to put this. To be consistent with how the other informers are handled, this should be in the location where the informer gets created (i.e. in csi-provisioner.go), not where it is used.

Reply:
This is the place it was placed: https://github.com/kubernetes-csi/external-provisioner/pull/590/commits/ff439eeba754c8dcbe4a4a1b9ed59ace51291133

Reply:
According to this commit, if this line was removed directly, the node topology worker will not run. If you change the node label or csinode, nothing will happen. It will cause problem

Reply:
I agree that removal of `go c.topologyInformer.Run(ctx)` in that commit introduced a bug. The question just is where that `Run()` call should go instead. I propose `cmd/csi-provisioner/csi-provisioner.go`.

Reply:
In fact, `c.topologyInformer.Run(ctx)` is not an informer. It is only a topology queue worker. the node informer and csinode informer are started by `factory.Start()` in `cmd/csi-provisioner/csi-provisioner.go`. Moreover, only capacity feature uses this worker and it should belong to capacity controller right?

Reply:
Maybe I can change this method name to avoid ambiguity like changing `Run` to `Run worker`

Reply:
Yes, `c.topologyInformer` it is not a `SharedInformer` instance from an informer factory. But conceptually it is the same.

We can rename `Run -> RunWorker` to make that more obvious.

Reply:
@pohly I have revised to this way. Can you review again? Thanks!